charters/ingest-infrastructure/charter.md

```diff
@@ -0,0 +1,166 @@
+
+# Ingestion Service
+
+## Description
+
+The Data Coordination Platform (DCP) Ingestion Service provides a suite of services to facilitate the contribution of data and metadata to the Human Cell Atlas (HCA). The Ingestion Service allows users to:
+  * Upload metadata describing biological materials, data files, and the experimental processes that generated them
+  * Validate metadata against endorsed community standards
+  * Interact with the DCP Upload Service to track and validate uploaded data files
+  * Track submission states that describe the life-cycle of experimental data as it moves through the DCP
+  * Generate identifiers for data, metadata and submissions
+  * Store submitted data in the DCP Data Storage Service
+
+### Definitions
+
+*Data contributor* - any user who uploads data into the DCP
+
+*Data consumer* - any user who accesses or downloads data from the DCP
+
+*Direct data contributor* - a data contributor who uploads their own data through services offered as part of the DCP in a "self-service" manner. Direct data contributors are also usually the generators of the data
+
+*Data wrangler* - a data contributor who works as part of the HCA consortium to provide assistance to biologists wanting to upload data into the DCP
```

Comment:
Could this be changed to 

Data wrangler - the Ingest Support team member who assists labs contributing data to the HCA. In the context of this charter, data wranglers considered as data contributors

or something along these lines

It is important to note that data wranglers will also support the direct data contributors in your parlance but they won't do as much for them.